server/index.ts

```diff
@@ -19,37 +18,34 @@ import type { FlashData, SessionData } from "./session";
 /** For some reason the globals like File only work on production build
  * In development, we need to install them manually
  */
-if (env.NODE_ENV !== "production") {
-  var webFetch = require("@remix-run/web-fetch");
-  global.File = webFetch.File;
-}
+// FIXME: look at this again
```

Comment:
@DonKoko any other cases I can test? Works on my machine 😂

Reply:
Not that I can think of. I will take a look around asap and let you know. So in vite it works without it?

Reply:
It seems yes. Tell me if you encounter any issue.

Reply:
I don't see the issue as well, so I think its good. Feel free to remove the commented code.

Reply:
done